raiden/transfer/architecture.py

```diff
@@ -132,6 +132,11 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
 
+class VerifiedSenderStateChange(StateChange):
```

Comment:
Why not authenticated? It has a clear definition and it's used broadly:

> Authentication is the process of verification that an individual, entity or website is who it claims to be.

https://www.owasp.org/index.php/Authentication_Cheat_Sheet#Introduction